src/parser/command.rs

```diff
@@ -205,17 +205,21 @@ pub(crate) fn parse_defcal_gate<'a>(
 pub(crate) fn parse_defcal_measure<'a>(
     input: ParserInput<'a>,
 ) -> InternalParserResult<'a, Instruction> {
-    let (input, qubit_index) = opt(token!(Integer(v)))(input)?;
-    let qubit = qubit_index.map(Qubit::Fixed);
-    let (input, destination) = token!(Identifier(v))(input)?;
+    let (input, (qubit, destination)) = map(
+        tuple((parse_qubit, opt(token!(Identifier(v))))),
```

Comment:
I believe this also exists, might be slightly faster:

```suggestion
        pair(parse_qubit, opt(token!(Identifier(v)))),
```

Reply:
Nice, done!